gdscript-docs.el

```diff
@@ -103,9 +107,8 @@ ORIG-FUN is function we wrap around.  ARGS are argument to ORIG-FUN function."
   "Convenience setup for pages with Godot documentation."
   (setq multi-isearch-next-buffer-function nil)
   (gdscript-docs--rename-eww-buffer)
-  (gdscript-docs--filter-content-to-main-div))
-
-(add-hook 'eww-after-render-hook #'gdscript-docs--eww-setup)
+  (gdscript-docs--filter-content-to-main-div)
+  (remove-hook 'eww-after-render-hook #'gdscript-docs--eww-setup))
```

Comment:
This removes the hook after rendering the first page, so if you browse from within eww, the unending menu won't be trimmed out.

I'm not familiar with this code, but isn't there a way to make the hook only valid for this eww buffer browsing godotengine.org?

Reply:
Yup, I should have spotted that. Rushed it. I'll address it tomorrow. Thanks.

Reply:
So I pushed an update, do I need to do anything else? Is this now flagged for you to review?

Reply:
Hmm. Not sure if this will work for local file based docs. I shall look into that.

Reply:
I've committed a fix for review that checks for docs.godotengine. I realised though that this won't work for local file-based docs. I shall look into that and update. 

Is the only download resource for HTML docs here: https://readthedocs.org/projects/godot/downloads/ ?

Reply:
I used wget to copy [doc.godotengine](https://docs.godotengine.org/en/stable/classes/index.html) and 

```
 '(gdscript-docs-local-path
   "/home/rgr/Documents/Programming/Godot/www/docs.godotengine.org/en/stable")

```

and `gdscripts-docs-browse-api `works fine.

`gdscripts-docs-browse-symbol-at-point `should also use local if available.

(readthedocs isout of date - not HTML docs for 3.2)

Reply:
I build the docs locally for this feature, with the pages pre-trimmed. The builds don't work indeed.

Reply:
The docs eco system seems very complicated ;) Anyway, `wget` worked for me and then local browsing worked fine.